tests/pkg_micro-ecc/tests/01-run.py

```diff
@@ -5,14 +5,13 @@
 
 
 # Use a custom global timeout for slow hardware. On microbit (nrf51), the
-# test completes in 80s.
-TIMEOUT = 100
+# test completes in 120s.
```

Comment:
The 120s are copied from the old pkg_micro-ecc-with-HWRNG test. This is probably bogus, because the test changed quite a bit, still better than 80s.